tests/conftest.py

```
@@ -126,6 +126,83 @@ def reset_realtime_state() -> None:
+        loggers += [
+            logging.getLogger(name)
+            for name in list(logging.root.manager.loggerDict)
+            if isinstance(logging.getLogger(name), logging.Logger)
+        ]
+        for lg in loggers:
+            for handler in list(getattr(lg, "handlers", ())):
+                stream = getattr(handler, "stream", None)
+                if stream is not None and getattr(stream, "closed", False):
+                    lg.removeHandler(handler)
+    except Exception:  # logging cleanup must never fail a test, least of all at teardown
+        pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)